sources/zafira-dbaccess/src/test/java/com/qaprosoft/zafira/dbaccess/dao/TestCaseMapperTest.java

```diff
@@ -82,10 +82,10 @@ public void getTestCaseById() {
         checkTestCase(testCaseMapper.getTestCaseById(TEST_CASE.getId()));
     }
 
-    @Test(enabled = ENABLED, dependsOnMethods = { "createTestCase" })
+    /*@Test(enabled = ENABLED, dependsOnMethods = { "createTestCase" })
     public void getTestCaseByClassAndMethod() {
         checkTestCase(testCaseMapper.getOwnedTestCase(TEST_CASE.getSecondaryOwner().getId(), TEST_CASE.getTestClass(), TEST_CASE.getTestMethod()));
-    }
```

Comment:
Is there any way to fix this test without commenting?

Reply:
@tsvrko these tests are deprecated, they don`t run along long time